src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -532,6 +542,13 @@ public static ImmutableArray<TypedConstant> Encode(TypeSymbol type, TypeSymbol s
                 return names;
             }
 
+            private static bool TryGetNames(TypeSymbol type, ArrayBuilder<string> namesBuilder)
+            {
+                type.VisitType((t, builder, _ignore) => AddNames(t, builder), namesBuilder);
+                Debug.Assert(namesBuilder.Any());
+                return namesBuilder.All(name => name == null);
```

Comment:
Yes, it will be more readable (the method name will make sense) by inverting the result and usage.